src/modules/Touchdown/pages/Dashboard.js

```diff
@@ -291,7 +291,11 @@ function Dashboard() {
                   <Button
                     buttonText="CREATE TOUCHDOWN"
                     className="fw-500 fs-16"
-                    onClick={() => navigate('/touchdown/contest')}
+                    onClick={() =>
+                      navigate('/touchdown/contest', {
+                        state: { fromDashboard: true },

```

Comment:
why we pass boolean , directly pass name that you used in , eg.  'touchdown' , so dont need to add condition there